### ISSUE TITLE
Introducing Catalogs for Small Businesses

### ISSUE THREAD

**Issue by marcelocecin** [open-wa/wa-automate-nodejs#40]
is possible to create a Message of type 'product' ?

[https://github.com/Rhymen/go-whatsapp/blob/master/binary/proto/def.proto](https://github.com/Rhymen/go-whatsapp/blob/master/binary/proto/def.proto)

[https://blog.whatsapp.com/10000665/Introducing-Catalogs-for-Small-Businesses](https://blog.whatsapp.com/10000665/Introducing-Catalogs-for-Small-Businesses)
[https://www.youtube.com/watch?v=ugFIVoh9IEQ](https://www.youtube.com/watch?v=ugFIVoh9IEQ)

**Comment by marcelocecin** [comment]
I believe the quickest and easiest way would be to create the product message on the mobile device, forward to some user and use that message as a default to forward to other contacts

**Comment by smashah** [comment]
@marcelocecin I don't know about creation of the product but I know how to send the product to a number. Good idea.

**Comment by marcelocecin** [comment]
can you share how to send the product ? thanks !

**Comment by smashah** [comment]
@marcelocecin Try this out.

https://github.com/smashah/sulla/commit/cb1712056842b4defe83a4ce6e118584119ec939

It's not in the client code yet.

**Comment by smashah** [comment]
@marcelocecin Untested. Try it out in v1.4.2 https://github.com/smashah/sulla/blob/a712ab638f732e85626896c16f4ee8c79f62fd8e/demo/index.ts#L47

**Comment by marcelocecin** [comment]
thanks, I will test it later today

**Comment by marcelocecin** [comment]
@smashah it did not work: 
error TypeError: Cannot read property '0' of undefined `products[0].id`

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.17](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.16...1.9.17) (2020-05-09)

- Release 1.9.17 [`2b88a8a`](https://github.com/open-wa/wa-automate-nodejs/commit/2b88a8af1798a9bc8baf8381bf41b04853b0d2eb)
- fix: business product catalog now working #40 [`00acc6e`](https://github.com/open-wa/wa-automate-nodejs/commit/00acc6ebf85f373e213275ec3abdd6c739b4b56c)
- updated release-image [`d51acc8`](https://github.com/open-wa/wa-automate-nodejs/commit/d51acc8ffab312ea39a25feb15a6f836112dc5fa)
